optoforce/config/SR_OMD45FE1000N.yaml

```diff
@@ -0,0 +1,6 @@
+counts_n_fx: 71.07
```

Comment:
And maybe this file should be renamed to something generic before we can merge it. Something like `single_channel_6_axis_generic_scale.yaml`.